### PR TITLE
Add prompt to facility form

### PR DESCRIPTION
**Pull request by antonbogatyrov**
## Related Issue or Background Info

- #1152 

## Changes Proposed

- When user tries to navigate away from the facility form without saving changes, they should get a warning

### CODE REVIEW

File: frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx [hunk 1]
```diff
@@ -26,110 +29,118 @@ const FacilityInformation: React.FC<Props> = ({
...
-        value={facility.name}
-        required
-        onChange={onChange}
-        onBlur={() => {
-          validateField("name");
-        }}
-        validationStatus={errors.name ? "error" : undefined}
-        errorMessage={errors.name}
+    <>
+      <Prompt
```
Comment (timothybest-usds) [inline frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx:33]: I believe this prompt should be in the `FacilityForm` component not the `FacilityInformation` as it should be at the form level and `FacilityInformation` is a set of inputs that is reused across multiple forms

**Review by timothybest-usds — CHANGES_REQUESTED** [review]

**Comment by maya** [comment]
In the prompt, do we have the ability to change "says" to something else? @timothybest-usds @antonbogatyrov

File: frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx [hunk 2]
```diff
@@ -26,110 +29,118 @@ const FacilityInformation: React.FC<Props> = ({
...
+        when={formChanged}
+        message={() =>
+          "\nYour changes are not yet saved!\n\nClick OK discard changes, Cancel to continue editing."
```
Comment (maya) [inline frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx:36]:
```suggestion
          "\nYour changes are not saved yet!\n\nClick OK to delete your answers and leave, or Cancel to return and save your progress."
```

**Review by maya — CHANGES_REQUESTED** [review]

**Comment by timothybest-usds** [comment]
> In the prompt, do we have the ability to change "says" to something else? @timothybest-usds @antonbogatyrov

Customizing the alert box in not possible. Note it does look different in each browser. Here is the alert in Firefox
Screen Shot 2021-04-12 at 10 18 22 AM

File: frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx (hunk 2 above)
Comment (antonbogatyrov) [reply frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx]: fixed

File: frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx (hunk 1 above)
Comment (antonbogatyrov) [reply frontend/src/app/Settings/Facility/Components/FacilityInformation.tsx]: done

**Review by maya — APPROVED** [review]
Alert message LGTM. 

I didn't see anything else frontend-related, but if there is, then let me know.

**Comment by antonbogatyrov** [comment]
@TimBest , @maya can i merge?

**Review by timothybest-usds — APPROVED** [review]

**Pull request merged**